pulsar-testclient/src/main/java/org/apache/pulsar/testclient/PerformanceProducer.java

```diff
@@ -785,7 +787,9 @@ private static void runProducer(int producerId,
                                     });
                         } else {
                             transaction.abort().thenRun(() -> {
-                                log.info("Abort transaction {}", transaction.getTxnID().toString());
+                                if (log.isDebugEnabled()) {
+                                    log.debug("Abort transaction {}", transaction.getTxnID().toString());
+                                }
```

Comment:
Please check the `exceptionally` method, the error log should be `Abort transaction`.

Reply:
@gaoran10 Done.